<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInteractorStyleWindowLevelThreshold</class>
 <widget class="QWidget" name="QInteractorStyleWindowLevelThreshold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QInteractorStyleWindowLevelThreshold</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Window level threshold</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="activeLabelLabel">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="text">
           <string>Threshold Label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>No label</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/transparentbox.png</normaloff>:/icons/transparentbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vessel Wall</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/bluebox.png</normaloff>:/icons/bluebox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lumen</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/redbox.png</normaloff>:/icons/redbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Calcification</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/greenbox.png</normaloff>:/icons/greenbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hemorrhage</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/yellowbox.png</normaloff>:/icons/yellowbox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LRNC</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/lightbluebox.png</normaloff>:/icons/lightbluebox.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LM</string>
           </property>
           <property name="icon">
            <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
             <normaloff>:/icons/purplebox.png</normaloff>:/icons/purplebox.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_62">
          <property name="text">
           <string>Targe Image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxTargeImage"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonThreshold">
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Main/resources/IADE_Analyzer.qrc"/>
 </resources>
 <connections/>
</ui>
